guides/learning_guide.rst

```diff
@@ -0,0 +1,1099 @@
+to coordinate with another task within the same workflow (e.g., in situ analysis) it
+must often resort to solutions with many side effects to overcome this barrier.
+
+With the innovation of Flux, jobs or tasks can connect to one another through
+messaging overlays and datastores built directly into Flux - a feature that significantly
+facilitates communication between jobs, breaking down the coordination barrier. In
+addition, Flux lets users configure and monitor all jobs and instances related to their
+workflow through the command-line and programming interfaces.
+
+Figure 2 visually contrasts the complexity of the conventional and new or emerging
+HPC paradigms in terms of how they utilize the resources allocated by the system's 
+workload manager. Nearly all existing products were designed when the workflows
+were much simpler as shown in Figure 2a. Yet, these solutions have significant
+difficulties with emerging paradigms exemplified by Figure 2b, as well as with
+connecting and coordinating jobs. These problems have led users to develop their
+own ad hoc custom scheduling and resource management software or use tools that
+perform only workflow management or only scheduling. However, accomplishing 
+sufficient job coordination without first-class support from a workload manager
+like Flux has already proven to be difficult for either approach. Perhaps more
+importantly, developing and maintaining ad hoc management software—especially
+
+
+.. figure:: images/figure-2.png
+   :alt: Different resource allocations between conventional and emerging scientific workflows
+   :align: center
+
+   An illustration of how the computing resources allocated to a job -- as granted by the HPC
+   workload manager -- differs between conventional and emerging scientific workflows. The
+   national x-axes depict the compute node IDs allocated to the job and the y-axes depict the IDs of
+   computing resources in each of these nodes. (a) The conventional paradigm requires only a single
+   parallel simulation application to run. (b) The emerging paradigm often requires many different
+   types of tasks such as ensemble of molecular dynamics (MD) parallel simulation applications
+   and another ensemble of docking simulation applications along with in suti data anlsyis for the
+   MD ensemble while these tasks are driven by an AI.
+   
+in this era of ever-evolving HPC hardware architectures—has quickly become
+prohibitively expensive for supercomputing application teams.
+
+Illustrative Batch Script Examples
+==================================
+
+In a job script used by a simpler workflow, the user enters line 1 in Figure 3a to
+submit a job script (line 2) to the system-level workload manager, requesting 256
+compute nodes with 42 CPU cores each. This job script uses the allocated resources
+under the conventional workflow paradigm (see Figure 2a), where the application
+(i.e., sim.app) is executed across 256 compute nodes simultaneously, which then
```

Comment:
should format `sim.app` in some way? (italicized seems wrong ... some type of "this is a command" formatting?)  In other parts of this section too.

Reply:
Now I remember, the double bacticks "inline literal" is what I was thinking.